pkg/resolvers/grbClusterResolver_test.go

```diff
@@ -34,6 +41,9 @@ func TestGRBClusterRuleResolver(t *testing.T) {
 
 func (g *GRBClusterRuleResolverSuite) SetupSuite() {
 	g.userInfo = NewUserInfo("test-user", "test-group")
+	g.broUserInfo = NewUserInfo("system:serviceaccount:cattle-resources-system:rancher-backup", "system:serviceaccounts")
```

Comment:
Should we test the same but with a different namespace? From what I see we're not looking at a namespace specifically.

Reply:
I don't think that's necessary - the code doesn't consider the namespace, so I'm not sure what value we gain by adding that test.

Reply:
I was thinking of adding it to prevent regressions. If we allow fleet in any namespace, then we should make sure that any namespace works in the future as well with the service account check, no? Wdyt?

Reply:
That logic could result in effectively infinite tests though. If I test `my-namespace-1`, what stops someone from adding a check that works for `my-namespace-1` and `cattle-local-system` but not `my-namespace-2`?